src/chainparams.cpp

```diff
@@ -417,8 +417,9 @@ class CTestNetParams : public CChainParams {
         consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nActivationHeight = 1028500;
         consensus.vUpgrades[Consensus::UPGRADE_CANOPY].hashActivationBlock =
             uint256S("01a4d7c6aada30c87762c1bf33fff5df7266b1fd7616bfdb5227fa59bd79e7a2");
-        consensus.vUpgrades[Consensus::UPGRADE_NU5].nProtocolVersion = 170015;
-        consensus.vUpgrades[Consensus::UPGRADE_NU5].nActivationHeight = 1599200;
+        consensus.vUpgrades[Consensus::UPGRADE_NU5].nProtocolVersion = 170050;
+        consensus.vUpgrades[Consensus::UPGRADE_NU5].nActivationHeight =
+            Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT;
```

Comment:
This needs to be set to the new testnet NU5 activation height before the PR is merged.